### PR TITLE
My part of the task

### PR DESCRIPTION
**Pull request by eutampieri**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Gradle Build CI
+name: Test using Gradle
```
Comment (Corstor) [inline .github/workflows/ci.yml:2]: Remove empty lines

Comment (Corstor) [inline .github/workflows/ci.yml]: Delete empty lines

File: build.gradle.kts [hunk 2]
```diff
@@ -1,3 +1,8 @@
 plugins {
-    java
+	java
+	application
+}
+
+application {
+    mainClass = "HelloTampieri"
```
Comment (Corstor) [inline build.gradle.kts]: Change to HelloStortini

**Review by Corstor — CHANGES_REQUESTED** [review]
Please

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Ventus218) [reply .github/workflows/ci.yml]: i think it's a bad idea

Comment (eutampieri) [reply .github/workflows/ci.yml]: Done

Comment (Ventus218) [reply .github/workflows/ci.yml]: i prefer eugenio way

File: build.gradle.kts (hunk 2 above)
Comment (eutampieri) [reply build.gradle.kts]: Done

**Review by Corstor — APPROVED** [review]
LGTM

**Pull request merged**